ChangeLog

```diff
@@ -11,6 +11,11 @@ Release date: TBA
 ..
   Put new features here and also in 'doc/whatsnew/2.12.rst'
 
+* Fix ``unused-private-member`` false positive using private methods within the class without
+  using it as a dot.method.
```

Comment:
```suggestion
* Fix ``unused-private-member`` false positive when accessing private methods through `property`
```

is this better? @DanielNoord

Reply:
Yes! Note that this should also be changed in the other changelog file.